android/src/main/java/com/mapbox/mapboxgl/MapboxMapsPlugin.java

```diff
@@ -86,110 +86,6 @@ public void onDetachedFromActivity() {
     lifecycle = null;
   }
 
-  // Old Plugin APIs
-
-  public static void registerWith(Registrar registrar) {
```

Comment:
This was not used, but please again if we could really drop that.

Reply:
That should be fine

Reply:
I just checked, because I was curious: the new APIs were introduced in Flutter 1.12, so this should indeed be safe to remove as you noted. ([https://docs.flutter.dev/release/breaking-changes/plugin-api-migration](https://docs.flutter.dev/release/breaking-changes/plugin-api-migration))
Maybe we could still add a note to the changelog if we are already documenting the new min android sdk.